client/src/main/kotlin/no/nav/poao_tilgang/client/PoaoTilgangHttpClient.kt

```diff
@@ -177,6 +174,22 @@ class PoaoTilgangHttpClient(
 				),
 				policyId = PolicyId.NAV_ANSATT_TILGANG_TIL_NAV_ENHET_V1
 			)
+
+			is NavAnsattBehandleStrengtFortroligBrukerePolicyInput -> PolicyEvaluationRequestDto(
+				requestId = policyRequest.requestId,
+				policyInput = NavAnsattBehandleStrengtFortroligBrukerePolicyInputV1Dto(
+					navAnsattAzureId = policyRequest.policyInput.navAnsattAzureId
+				),
+				policyId = PolicyId.NAV_ANSATT_BEHANDLE_STRENGT_FORTROLIG_BRUKERE
+			)
+
+			is NavAnsattBehandleFortroligBrukerePolicyInput -> PolicyEvaluationRequestDto(
+				requestId = policyRequest.requestId,
+				policyInput = NavAnsattBehandleFortroligBrukerePolicyInputV1Dto(
+					navAnsattAzureId = policyRequest.policyInput.navAnsattAzureId
+				),
+				policyId = PolicyId.NAV_ANSATT_BEHANDLE_FORTROLIG_BRUKERE
+			)
```

Comment:
Tenker det kan være greit å ha tester for hver av disse i PoaoTilgangHttpClientTest.kt

Reply:
@AlexGaard takk for din review. Tests er oppdatert. Kan du sjekke igjen, er du snill.